client/wayland/ibuswaylandim.c

```diff
@@ -1316,17 +1743,35 @@ registry_handle_global (void               *data,
         fprintf (priv->log, "wl_reistry gets interface: %s\n", interface);
         fflush (priv->log);
     }
```

Comment:
Not changed in this PR but a in the `fprintf` call a few lines up there is a typo: `s/wl_reistry/wl_registry/`